python/amici/ode_export.py

```diff
@@ -714,11 +715,12 @@ class ODEModel:
         derivative from a partial derivative call to enforce a partial
         derivative in the next recursion. prevents infinite recursion
 
-        _simplify: if set to true, symbolic derivative expressions will be
-        simplified. NOTE: This does currently not work with PySB symbols.
+        _simplify: if set, this function will be used to simplify symbolic
+         derivative expressions.
+         NOTE: This does currently not work with PySB symbols.
     """
 
-    def __init__(self, simplify: bool = False):
+    def __init__(self, simplify: Callable = sp.powsimp):
```

Comment:
can we make simplify an iterable of Callables that are to be applied sequentially? Otherwise its not straightforward to me what to pass if no simplifications are to be applied.

Reply:
Can do that, or change typehint to `Optional[Callable]` which implies None as a valid option. Do you see clear applications for an Iterable? Can also use None as default here.

Reply:
What if I want to apply both `ratsimp` and `powsimp`? For the models we look at we actually apply multiple rounds of simplifications step to specific expressions in the model (this is done before model import, so it doesnt need to be applied here, but should illustrate that this is sometimes reasonable).

Reply:
The you pass a lambda doing that ;)

Reply:
Oh passing lambda is probably the better approach as its overall more flexible. Fine with me to just mention that in the documentation.